ui/themes/app.js

```diff
@@ -9,6 +9,8 @@ export const Colors = {
   caribbeanGreen: '#00D3a9',
   keppelGreen: '#00B39F',
   charcoal: '#3C494F',
+  charcoal_100: '#fdfdfd',
```

Comment:
why adding new colors, there are already many existing theme colors and they are used for input fields, Please try to use them instead creating new ones

Reply:
Sure, I'll do this and create a new PR again. I only did this as it's mentioned in the issue to use sistent theme colors, and I couldnt find any sistent theme colors that are white and named according to the documentation of sistent.

Reply:
@nebula-aac , do we have this color and theme documented?